packages/sdk/src/oracle/README.md

```diff
@@ -12,14 +12,25 @@ import {oracle} from '@uma/sdk'
 // create your configuration object with known addresses
 type Config = oracle.types.state.Config
 
-// creating a config for mainnet usage add any additional chains you want
+// creating a config for mainnet usage add any additional chains you want following the same format
 export const config = {
   chains: {
     1:{
       chainId: 1,
-      multicall2Address: process.env.multicall2Address,
-      optimisticOracleAddress: process.env.optimisticOracleAddress,
+      multicall2Address: "0x5BA1e12693Dc8F9c48aAD8770482f4739bEeD696",
```

Comment:
if its not syncronous it might be wierd, since theres no way to async instantiate any of this from the frontend, it currently relies on this behavior. if it is syncronous then yea we could provide some defaults here!

Reply:
i added this change in, unfortunately it wasnt very easy. because node and frontend have different interfaces to get address (one async, one not) it forces it to either work on one or the other.  by defaulting to only using contracts-frontent, the library fails to import in node, and using contracts-node, it forces the default functions to be async.  I ended up copying just the functions i needed into the utility file to get this working on both FE and in e2e tests.